utils/dynamicLink.js

```diff
@@ -138,12 +138,23 @@ const generateCommunityLink = async (communityName) => {
     }
 }
 
+const generateUsernameLink = async (username) => {
+    try {
+        const betterWebAppUrl = `${BETTER_WEB_APP_URL}/?username=${username}`
+        return await generateLongDynamicLink(betterWebAppUrl)
+    } catch(e) {
+        console.log(e);
+        return false;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `generateUsernameLink` function could be improved. Currently, it just logs the error and returns false. It would be better to throw the error so that the calling function can decide how to handle it. This way, we can avoid silent failures and make debugging easier.
```diff
-        console.log(e);
-        return false;
+        throw e;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->